src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCNV.java

```diff
@@ -116,7 +116,7 @@
             shortName = ExomeStandardArgumentDefinitions.ALLELIC_PON_FILE_SHORT_NAME,
             optional = true
     )
-    protected File allelicPONFile;
+    protected File allelicPoNFile;
```

Comment:
You're right.  We have been a little lax on following the camel-case conventions exactly, but no time like the present to start being more strict!